src/ring_theory/hahn_series.lean

```diff
@@ -394,6 +394,9 @@ lemma sub_coeff' {x y : hahn_series Γ R} :
 lemma sub_coeff {x y : hahn_series Γ R} {a : Γ} :
   (x - y).coeff a = x.coeff a - y.coeff a := by simp
 
+lemma neg_order [has_zero Γ] {f : hahn_series Γ R} : (- f).order = f.order :=
```

Comment:
This should probably be renamed to `order_neg` to match e.g. `order_one`, `order_mul`:
```suggestion
lemma order_neg [has_zero Γ] {f : hahn_series Γ R} : (- f).order = f.order :=
```

Reply:
Actually, I came up with one further suggestion. Sorry for the many separate comments!
```suggestion
@[simp]
lemma order_neg [has_zero Γ] {f : hahn_series Γ R} : (- f).order = f.order :=
```

Reply:
I agree with @Vierkantor here that removing the `-` with `simp` is a good thing.